src/EventStore.Common/Configuration/ConfigurationBuilderExtensions.cs

```diff
@@ -28,5 +33,59 @@ public static class ConfigurationBuilderExtensions {
 
 			return builder;
 		}
+
+		// Allows configuration to be mounted inside a specified section
+		public static IConfigurationBuilder AddSection(
+			this IConfigurationBuilder self,
+			string sectionName,
+			Action<IConfigurationBuilder> configure) =>
+			
+			self.Add(new SectionSource(sectionName, configure));
+
+		public static IConfigurationBuilder AddConfigFile(
```

Comment:
Consider changing the name of the function to indicate its behaviour is specific to ESDB.